docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<p>Maybe get rid of <code>return_inferencedata</code> here, as it's False, and introduce it below, as you set it to True?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>I wanted it to be future-proof, given that with this PR, not passing it raises a FutureWarning...</p><p> </p><p>This example actually uses the <code>MultiTrace</code> API to access transformed variable samples.</p>

Reply:
Ah you're right, let's keep it to `False` explicitly then 👌